adapters/adoppler/adoppler.go

```diff
@@ -172,6 +181,18 @@ func (ads *AdopplerAdapter) MakeBids(
 	return adsResp, nil
 }
 
+func (ads *AdopplerAdapter) bidUri(ext *openrtb_ext.ExtImpAdoppler) (string, error) {
+	params := macros.EndpointTemplateParams{}
+	params.AdUnit = url.PathEscape(ext.AdUnit)
+	if ext.Client == "" {
+		params.AccountID = DefaultClient
+	} else {
+		params.AccountID = ext.Client
```

Comment:
Is this freeform text? If so, please consider running it through a UrlEscape call.